projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -388,7 +380,7 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
             return true;
         }
         const searchValue = this.searchValue || this.comboInput.value;
-        return !!searchValue && value.toString().toLowerCase().includes(searchValue.trim().toLowerCase());
+        return !!searchValue && value.toString().toLowerCase().includes(searchValue.toLowerCase());
```

Comment:
In general search implementations trim user input before searching. Think this trim might have been removed accidently.